### DISCUSSION TITLE
Startup on Github

### DISCUSSION THREAD

**Discussion by jurmy24** [community/community#21552] [New to GitHub]
<p>What would be the <span class="hashtag">#1</span> advice for a newbie to github to gain experience fast?</p>

**Comment by Kukun7610** [comment, marked as answer]
<p><strong>You’ll learn what is:</strong></p>
<p><strong>  1. repository</strong></p>
<p><strong>  2. Branch</strong></p>
<p><strong>  3. how can u upload ur projects</strong></p>
<p><strong>  4. How to contribute other projects </strong></p>
<p><strong>for more plz visit <a href="https://guides.github.com/activities/hello-world/" rel="nofollow noopener">https://guides.github.com/activities/hello-world/</a> very familiar article by <span class="mention">@githubTeam</span></strong></p>
<p>Hope these help! </p>
<p>Happy Coding!</p>

**Comment by geert564** [comment]
With the world's largest software registry, GitHub makes it simple to find the best open source projects to accelerate your workflows and scale your business fast. There is a huge open source community of code, [Säkerhetsdörrar](https://branddorrar.com/sakerhetsdorrar/?fbclid=IwAR0FYOmZBNJ1sYIvXRTpmDbS69dCmjy1dsIPzTyDH5B1WPrK_CxbUiqtC8U) and developers to work with and draw inspiration from on GitHub.
